cobra/core/model.py

```diff
@@ -556,68 +693,6 @@ def optimize(self, objective_sense='maximize', solution_type=Solution,
             #         self, solution.status))
         return solution
 
-    def remove_reactions(self, reactions, delete=True,
-                         remove_orphans=False):
-        """remove reactions from the model
-
-        reactions: [:class:`~cobra.core.Reaction.Reaction`] or [str]
-            The reactions (or their id's) to remove
-
-        delete: Boolean
-            Whether or not the reactions should be deleted after removal.
-            If the reactions are not deleted, those objects will be
-            recreated with new metabolite and gene objects.
-
-        remove_orphans: Boolean
-            Remove orphaned genes and metabolites from the model as well
-
-        """
-        if isinstance(reactions, string_types) or hasattr(reactions, "id"):
-            warn("need to pass in a list")
-            reactions = [reactions]
-        for reaction in reactions:
-            try:
-                reaction = self.reactions[self.reactions.index(reaction)]
-            except ValueError:
-                warn('%s not in %s' % (reaction, self))
-            else:
-                if delete:
-                    reaction.delete(remove_orphans=remove_orphans)
-                else:
-                    reaction.remove_from_model(remove_orphans=remove_orphans)
-
-    def repair(self, rebuild_index=True, rebuild_relationships=True):
```

Comment:
I don't know why this repair change is getting wrapped up with the remove reactions change... No intention to edit here